app/pb_kits/playbook/pb_button_toolbar/docs/_button_toolbar_default.jsx

```diff
@@ -1,25 +1,58 @@
 import React from 'react'
 import { Button, ButtonToolbar } from '../..'
 
-const ButtonToolbarDefault = () => (
-  <div>
+const ButtonToolbarDefault = (props) => (
+  <div className="pb--doc-demo-row">
     <ButtonToolbar
+        {...props}
+        connected
+        orientation="vertical"
+    >
+      <Button
+          {...props}
+          text="Field"
+      />
+      <Button
+          {...props}
+          text="Retail"
+      />
+      <Button
+          {...props}
+          text="Event"
+      />
+      <Button
+          {...props}
+          text="Training"
+      />
+      <Button
+          {...props}
+          text="Not Working"
+      />
```

Comment:
Can we think of other examples for the button text? These are very power specific, and also I think we can get creative here. See the nav examples for inspiration